### ISSUE TITLE
Feature request: Seamless transition when spectating 3rd person and going into freecam

### ISSUE THREAD

**Issue by BerserkerBG** [stijn-volckaert/udemo#73]
When you are spectating someone in 3rd person and you decide to go into free cam to fly around map, it will remember your previous freecam position on the map and spawn you there. Here is a quick example with normal 469 demo manager:

https://github.com/stijn-volckaert/udemo/assets/58362332/eba6ebaa-cdf0-435b-93e4-bce194d58385

Here is how it looks like with Movie Demo Manager (works only on 436):

https://github.com/stijn-volckaert/udemo/assets/58362332/762c0ddd-bab6-4881-a6c7-f8f0937f59fc

Basically second example is my proposal. When you exit 3rd person it needs to place you next to the player you were spectating.

**Comment by SeriousBuggie** [comment]
Some mods/users can rely on current implementation. And it is like current spectator work in game.
So I suggest add special button - for move actual spectator to camera position. And who need add it to bind.

Currently you can how click by both mouses for move spectator to close location. but this work for next player, not current.

**Comment by BerserkerBG** [comment]
> Some mods/users can rely on current implementation. And it is like current spectator work in game. So I suggest add special button - for move actual spectator to camera position. And who need add it to bind.
> 
> Currently you can how click by both mouses for move spectator to close location. but this work for next player, not current.

What if its implemented for jump key only and leave the mouse buttons alone to default behavior? There are old mods that rely on moviedemomanager's feature and unfortunately its 436 exclusive.

**Comment by SeriousBuggie** [comment]
I think if it be separate command you can put it whenever you want. On key alone, On key together, or even on release key.

**Comment by SeriousBuggie** [comment]
Possible implementation:
New command 
`FollowMyCam 1`
`FollowMyCam 0`
By default follow enabled, but if change, it remembered in config variable FollowMyCam in udemo.ini.
So you not need input this command each time.
